internal/api/block_schemas.go

```diff
@@ -0,0 +1,48 @@
+package api
+
+import (
+	"context"
+
+	"github.com/google/uuid"
+)
+
+// BlockSchemaClient is a client for working with block schemas.
+type BlockSchemaClient interface {
+	List(ctx context.Context, blockTypeSlugs []uuid.UUIDs) ([]*BlockSchema, error)
+}
+
+// BlockSchema is a representation of a block schema.
+type BlockSchema struct {
+	BaseModel
+	BlockType
+
+	Checksum     string      `json:"checksum"`
+	BlockTypeID  uuid.UUID   `json:"block_type_id"`
+	Capabilities []string    `json:"capabilities"`
+	Version      string      `json:"version"`
+	Fields       interface{} `json:"fields"`
+}
+
+// BlockTypeID represents a block type ID.
+type BlockTypeID struct {
+	Any []uuid.UUIDs `json:"any_"`
+}
+
+// BlockSchemas represents a block schema.
+type BlockSchemas struct {
+	BlockTypeID
+}
```

Comment:
ooc, how are these guys getting used?

Reply:
Ah, meant to delete this since I used the nested struct instead. The nested struct was the only way I've seen so far to still provide struct tags (in this example, I'm not sure how I'd add a struct tag to `BlockTypeID`).

Fixed in 93fe963.